crawl-ref/source/stairs.cc

```diff
@@ -422,6 +422,22 @@ static void _new_level_amuses_xom(dungeon_feature_type feat,
     }
 }
 
+/**
+ * Determine destination level.
+ *
+ * @param how         How the player is trying to travel.
+ *                    (e.g. stairs, traps, portals, etc)
+ * @param whence      Not currently used
```

Comment:
this param is not used. should I remove it? or is it better to just leave it here. Since it's not used, I didn't know the best way to explain what it really is. I kept getting confused on how to explain the difference between 'how' and 'whence' .

Reply:
If it's unused then removing it sounds good - it might be worth doing a little code archaeology to figure out what (if anything) it was used for in the past, if possible.

Reply:
looks like this variable may have been there since that function was created in a big refactor in 2015 09e009602f11412cbc. it wasn't used then either.